onnxruntime/core/framework/sequential_executor.cc

```diff
@@ -302,7 +302,7 @@ Status SequentialExecutor::Execute(const SessionState& session_state, const std:
 #endif
 #ifdef ENABLE_NVTX_PROFILE
       profile::NvtxRangeCreator node_compute_range(
-          MakeString(node.OpType(), ".", node.Index()), profile::Color::Blue);
+          node.Name(), profile::Color::Blue);
```

Comment:
i think the OpType is also useful. how about something like `MakeString(node.OpType(), ".", node.Index(), "(", node.Name(), ")")`

Reply:
+1